lib/config.js

```diff
@@ -1,9 +1,9 @@
 const Joi = require('@hapi/joi');
-const clone = require('clone');
```

Comment:
@adamjmcgrath what is the reason for this clone in the first place? I can't remember.

@davidpatrick what is the reason for this deletion?

Reply:
I deleted it because I didn't want to clone the store instance (in case user's added event listeners to it etc.)

I believe it was there to prevent unwanted mutations to the user's config, but objects passed into joi are effectively immutable and mutating the store instance is desirable (adding event listeners etc.) - so I don't think it's necessary (I removed it for the Next SDK as well)